demos/spark/prepare_env.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Env variables
+container=intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-occlum
+tag=2.1.0-SNAPSHOT
+SPARK_HOME=/opt/spark
+
+# Install openjdk11
+apt-get update
+apt-get install -y openjdk-11-jre
```

Comment:
Is it necessary to install "openjdk-11-jre"?

Reply:
Yes, because the java-11-openjdk-amd64.yaml requires openjdk-11-jre. The az team uses openjdk-11 to run spark, so I bring into correspondence with their implementions.